### PR TITLE
leafy-logger

### PR DESCRIPTION
**Pull request by mkristian**
the README.md is not ready so have a look at https://github.com/lookout/leafy/blob/logger/leafy-logger/spec/factory_spec.rb#L9 on how to configure logger.

### CODE REVIEW

File: leafy-logger/lib/leafy/logger/factory.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+      def self.bootstrap
+        Java::IoDropwizardLogging::LoggingFactory.bootstrap
+      end
+
+      def initialize
+        @factory = Java::IoDropwizardLogging::LoggingFactory.new
+      end
+
+      def level args = nil
```
Comment (rtyler) [inline leafy-logger/lib/leafy/logger/factory.rb]: What did parenthesis do to you that makes you hate them so much :-P

**Comment by rtyler** [comment]
What does configuration look like? Also what's a bit unclear to me is how you might change the log-levels in gerneral or a specific class after code is already running

File: leafy-logger/lib/leafy/logger/factory.rb (hunk 1 above)
Comment (mkristian) [inline leafy-logger/lib/leafy/logger/factory.rb]: actually I do not know if I hate them
but I will see to have them consistently in the future  ;)

**Comment by mkristian** [comment]
https://github.com/lookout/leafy/blob/logger/leafy-logger/spec/factory_spec.rb#L65
https://github.com/lookout/leafy/blob/logger/leafy-logger/spec/factory_spec.rb#L37

both are methods on the LoggerFactory and whenever you change the log level there it will be propagated.

configuration is currently manually

but to add the usual dropwizard config from http://dropwizard.io/manual/core.html#logging is just a matter of pulling in the "config" library and get an entry point on ```Leafy::Logger::LoggerFactory.newFromYaml``` with a 2-3 lines of code.

**Comment by mkristian** [comment]
fixes #7 more or less

**Pull request merged**